### PR TITLE
[WIP] Alemata.angular2 quickstart

### PR DESCRIPTION
**Pull request by alemata**
### Your checklist for this pull request

Please review the [guidelines for contributing](https://github.com/auth0/docs#contributing) to this repository.
- [ ] Content conforms to our [Contributing Guidelines](https://github.com/auth0/docs#contributing-guidelines)
- [ ] If applicable, you have added details to the [update feed](https://github.com/auth0/docs/tree/master/updates) 

If you are an Auth0 employee, after you submit this PR create and test a [Review App](https://github.com/auth0/docs#review-apps) to test your changes.
### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

File: articles/client-platforms/angular2/00-intro.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+#### 2: Viewport
+
+If you are targeting mobile audiences, it's recommended that you add in `<head>` element:
+
+```html
+<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0"/>
+```
+
+Please continue with next [step](link_to_login) tutorial to know how to implement login using `Auth0`.
```
Comment (alemata) [inline articles/client-platforms/angular2/00-intro.md:70]: @ntotten I am not sure how to add this link

File: articles/client-platforms/angular2/03-user-profile.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+      <p><strong>Updated At: </strong> {{auth.userProfile.updated_at}}</p>
+    </div>
+  </div>
+</div>
+<h4 *ngIf="!auth.authenticated()">You are not logged in, please click 'Log in' button to login</h4>
+```
+
+#### Custom Sign Up Fields
+
+You can add input fields to the sign up form adding to the options parameter the `additionalSignUpFields`. See full documentation [here](link_to_additionalSignUpFields_doc).
```
Comment (alemata) [inline articles/client-platforms/angular2/03-user-profile.md:91]: @ntotten I am not sure how to add this link

File: articles/client-platforms/angular2/01-login.md [hunk 3]
```diff
@@ -3,91 +3,80 @@ title: Login
...
 
-### 1. Add the Auth0 Scripts and Install angular2-jwt
+// Avoid name not found warnings
+declare var Auth0Lock: any;
 
-Install **angular2-jwt** with **npm**.
+@Injectable()
+export class Auth {
+  // Configure Auth0
+  lock = new Auth0Lock('YOUR_CLIENT_ID', 'YOUR_DOMAIN', {});
```
Comment (alemata) [inline articles/client-platforms/angular2/01-login.md:26]: @ntotten should we use  `${account.clientId}` here?

Comment (ntotten) [inline articles/client-platforms/angular2/01-login.md:26]: yes

**Pull request merged**